docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,64 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** P-RMD each SWH Distribution
+**Data Lookup:**   
+**Function Call:** 
+- **get_component_by_id**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements
+- **get_SWH_equipment_associated_with_each_swh_distribution_system**  
+
+**Applicability Checks:**
+- check that the U_RMD has SWH Uses with loads
```

Comment:
I think this rule is needed because where there is no SWH designed P_RMD would not equal U_RMD. I wonder how this is handeled for analogous situations for HVAC...In the schema these HVAC components are set to TRUE for P_RMD equalling U_RMD. I think this is a good call discussion topic.